src/Controller/EventsController.php

```diff
@@ -203,4 +205,39 @@ protected function getEventInstances(
     return $instances;
   }
 
+  /**
+   * Calculated an inherited field from an event instance due to calculated
+   * field issues: https://www.drupal.org/project/drupal/issues/2962457
+   *
+   * @param int $id
+   *    The event instance entity id.
+   * @param string $field
+   *    The field to fetch from the event instance/series.
+   * @return array
+   */
+  public function getInheritedFieldLabels(int $id, string $field): array
+  {
+    // Get the field values from the parent series.
+    $series_values = \Drupal::service('entity_type.manager')
```

Comment:
Let's use Dependency Injection and avoid static calls for better container performance and cacheability